github-dark.css

```diff
@@ -1529,8 +1529,9 @@
   a.filter-item.selected, .date-selector td.selected,
   .date-selector td.selectable-day:hover, .subnav-item:hover, .subnav-item:focus,
   .owner-select-target:hover, .owner-select-target:focus,
-  .suggester li.navigation-focus, .suggester li.navigation-focus small {
-    color: #efefef !important;
+  .suggester li.navigation-focus, .suggester li.navigation-focus small,
+  .label-select-menu .select-menu-item.selected, .octicon .octicon-x {
```

Comment:
The second part might not even be necessary. If `.octicon` proves to have enough specificity, color will just cascade down to the SVG.